lib/dartino.dart

```diff
@@ -103,6 +105,48 @@ void openDartinoSettings([_]) {
   atom.workspace.open('atom://config/packages/dartino');
 }
 
+_createNewProject([_]) async {
+  var sdk = await findSdk(null);
+  if (sdk == null) return;
+
+  String projectName = 'stm32f746g_discovery_project';
```

Comment:
There's a very cool pub package that will generate random names. We use it in dartpad and for flutter; `haikunator`. FYI for future iterations.